flex-box-template-home.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="flex-box-template-style.css">
+    </head>
+    <body>
+        <div class=layout>            
+            <div class=header>
+                <div class=container>
+                    <nav class=nav>
+                        <div class=left-nav>
+                            <a href="flex-box-template-home.html">Melahat Tayli </a>
+                        </div>
+                        <div class=right-nav>
+                            <a class=right-nav-item href="flex-box-template-research.html">Research</a>
+                            <a  class=right-nav-item href="flex-box-template-software.html">Software</a>
+                        </div>      
+                    </nav>
+                                  
+                </div>
+            </div> 
+            <div class=main> 
+                <div class=container>
+                    <div class=content>
+                        <p> <img src="profile-pic.jpg" height="250px" style="padding-right:10px"></p>
+                        <div class=content-paragraph>
+                            <h2> Hey, I'm Melahat Tayli </h2>
+                            <p>I am a M.Sc student at Harvard University Extention school in the field of Data Science.</p>
+                            <p>In my previous life, I was a material scientist, specialized in photopolymerization. I worked as a researcher in <a href="https://www.pccl.at/en/"> Polymer Competence Center Leoben</a> and collaborated with <a href="https://www.sempermed.com/en/contact/medical-gloves/">Semperit Technische Produkte Gesellschaft m.b.H.</a></p>
+                            <p>I am currently raising my baby boy and learning html, css and javascript as a hobby. You can follow my projects on my <a href="https://github.com/melahat-tayli">github</a>.</p>
+                        </div>                        
+                    </div>               
+                </div>    
+            </div>            
+            <div class=footer>
+                <div class=container>
+                    <ul class="contact-list">
+                        <li>Melahat Tayli</li>
+                        <li><a href="sahinmelahat@hotmail.com">sahinmelahat@hotmail.com</a></li>
+                    </ul>
+                    <ul class="social-media-list">
+                        <li><a href="https://github.com/melahat-tayli">
```

Comment:
You should not use this structure `<li><a>`.
Correct way is 
`<li> 
       <a>`
for writing clean code. you can apply the same process to similar codes.  (excluding SVG and inside of SVG)